cogs/commandowner.py

```diff
@@ -72,6 +73,8 @@ async def startlog(self, ctx):
             await self.utils.log(f"Start to log users from Server:\n\t{guildName}", 2)
         else:
             await ctx.send("Bot is logging. Logging state: True")
+        # Sets the bot's presence to "Online" to indicate it's logging.
+        await self.bot.change_presence(status=discord.Status.online, activity=discord.Game(str(self.jh.getFromConfig("command_prefix")) + "help"))

```

Comment:
It is unnecessary to always set the status and activity.

Reply:
If I set only one of them, the other is missing nevertheless the status game activity was there before. Testet this multiple times.

Reply:
This is not what I meant. For example, when calling the function `startlog` twice without any logging state change in between, the presence of the bot is updated twice, which is not needed, since the presence should already be correct.

Reply:
Got it. Will fix